mirrord/agent/src/entrypoint.rs

```diff
@@ -454,15 +453,19 @@ impl ClientConnectionHandler {
                 ))
                 .await?;
             }
-            ClientMessage::SwitchProtocolVersion(version) => {
+            ClientMessage::SwitchProtocolVersion(client_version) => {
+                let our_version = mirrord_protocol::VERSION.get().clone();
```

Comment:
Need also to have some initial value of `1.2.1` as `protocol_version` variable above

Reply:
Also don't need get it's a LazyLock

Reply:
got it - I think we can remove it since the switch protocol exists for 10 months already

Reply:
😅

Reply:
I'm just afraid that having a base which is a very "old" version while none actually uses it (according to metrics) will create hard bugs (why is it this version??)